src/Purestone/Card.hs

```diff
@@ -4,13 +4,16 @@
 -- This source code is licensed under the MIT licence found in the           --
 -- LICENSE file in the root directory of this source tree.                   --
 -------------------------------------------------------------------------------
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
I think it might be best to include this as a default extension in the `package.yaml` file:

```
default-extensions:
- DeriveGeneric
```
rather than have it at the top of every file